### PR TITLE
[14.0][ADD] stock_quant_reservation_info_mrp

### PR DESCRIPTION
**Pull request by DavidJForgeFlow**
Allows to see the manufacturing order related to the reserved info of Products.

@ForgeFlow

### CODE REVIEW

File: stock_quant_reservation_info_mrp/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Copyright 2019-2020 ForgeFlow S.L.
```
Comment (LoisRForgeFlow) [inline stock_quant_reservation_info_mrp/__manifest__.py]: 2022

**Review by LoisRForgeFlow — APPROVED** [review]
Functional test :+1:

File: stock_quant_reservation_info_mrp/__manifest__.py (hunk 1 above)
Comment (DavidJForgeFlow) [reply stock_quant_reservation_info_mrp/__manifest__.py]: Done!

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request merged**